content/community/support.md

```diff
@@ -29,7 +29,7 @@ Each community consists of many thousands of React users.
 * [Reactiflux online chat](https://discord.gg/reactiflux)
 * [Reddit's React community](https://www.reddit.com/r/reactjs/)
 * [Spectrum's React community](https://spectrum.chat/react)
-
+* [Discords react community](https://discord.com/invite/reactiflux)
```

Comment:
We already link to it with "Reactiflux online chat". Maybe rename the existing link to "Reactiflux discord community" instead?

Reply:
uh sure